### DISCUSSION TITLE
[Esame 13 febbraio 2024] Esercizio 4

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#449] [Esami M-Z]
![13 feb es.4](https://github.com/user-attachments/assets/21db2cc1-e32d-4788-a2fe-f20f53295b75)

**Comment by AglaiaNorza** [comment]
Soluzione del punto 1 ad opera della tutor [Elisa Scandiuzzi](https://github.com/sapienzastudentsnetwork/algebra/blob/main/scandiuzzi/README.md), lezione del 22 ottobre 2024 (`appunti_24_10_22.pdf`):

![1](https://github.com/user-attachments/assets/b76e79fb-2d50-4c4b-8e2c-310d72ab4c3d)
![2](https://github.com/user-attachments/assets/819b4946-5d3e-4ff5-bc23-ee769a3cd046)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/94039482-1c55-471f-bd37-da07a77129ef)

**Comment by niccolo-fato** [comment]
<img width="1280" height="1026" alt="image" src="https://github.com/user-attachments/assets/d16c3ab7-3c9f-411c-886b-c8a0827da6d3" />
